vnext/Desktop.IntegrationTests/HttpResourceIntegrationTests.cpp

```diff
@@ -17,7 +17,9 @@ TEST_CLASS (HttpResourceIntegrationTest) {
     Assert::IsFalse(nullptr == rc);
   }
 
-  TEST_METHOD(RequestGetSucceeds) {
+  // This test always fails because the requested resource does not exist.
+  // TEST_METHOD(RequestGetSucceeds) {
```

Comment:
>// TEST_METHOD [](start = 2, length = 14)

Does whatever harness this project uses support a SKIPTESTMETHOD annotation?

Reply:
No, this is VS test. We cannot use the SKIPTESTMETHOD here.

---
In reply to: [433405820](https://github.com/microsoft/react-native-windows/pull/5071#discussion_r433405820) [](ancestors = 433405820)